### PR TITLE
schema: Move generated schema out of tree

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

/cc @0xFelix 
/help

Carrying the generated KubeVirt JSON schema in tree is getting cumbersome. This change moves it outside and ensuring it is fetched before kubeconform is called to validate our generated resources.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 .idea
 _cluster-up/
 _kubevirt/
+_schemas/main/*
```
Comment (0xFelix) [inline .gitignore:6]: Change that to `_schemas/`?

Comment (lyarwood) [reply .gitignore:6]: @0xFelix  I can add a comment here echoing what I wrote in the commit but I only wanted to ignore the `main` branch in the cache as it's likely to change so often. For the more stable release branches I thought we could continue using the cache as things are less likely to change. Thoughts?

Comment (0xFelix) [reply .gitignore:6]: Makes sense, I missed the comment in the commit message, sorry.

**Pull request merged**